platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_general.adoc

```diff
@@ -175,3 +175,206 @@ include::sample[dir="snippets/bestPractices/avoidInternal/groovy",files="build.g
 === Tags
 
 <<tags_reference.adoc#tag:upgrades,`#upgrades`>>
+
+[[modularize_builds]]
+== Modularize Your Builds
+:keywords: structuring-builds
+
+Modularize your builds by splitting your code into subprojects.
```

Comment:
In Gradle context `subprojects` imply children of a given project (https://docs.gradle.org/nightly/javadoc/org/gradle/api/Project.html#getSubprojects())

Given that, should we just say "multiple projects"?

Because I don't think the goal of this practice to put split up projects under a specific project hierarchy.